docs/logs/correlation/Program.cs

```diff
@@ -13,33 +13,23 @@ public class Program
 
     public static void Main()
     {
-        var tracerProvider = Sdk.CreateTracerProviderBuilder()
-            .AddSource("MyCompany.MyProduct.MyLibrary")
-            .AddConsoleExporter()
-            .Build();
-
-        var loggerFactory = LoggerFactory.Create(builder =>
-        {
-            builder.AddOpenTelemetry(logging =>
+        var sdk = OpenTelemetrySdk.Create(builder => builder
```

Comment:
keep existing code as-is. If there is a need to move them to new format, they can be done in separate PR.